src/app/extensions.h

```diff
@@ -30,6 +30,7 @@ namespace app {
   class Extensions;
 
   struct ExtensionInfo {
+    std::string themeId;
     std::string name;
     std::string version;
     std::string dstPath;
```

Comment:
Use a bool just to indicate that this extension info contains one theme that tries to overwrite the default one:
```suggestion
    std::string name;
    std::string version;
    std::string dstPath;
    bool defaultTheme = false;
```